text2emoji.py

```diff
@@ -1,20 +1,46 @@
+def main():
 # the dictionary that stores the character and their emoji counterpart
-convert_dict = {}
+    convert_dict = {}
 
 # ASCII 'A' is 65
-counter = 65
-for emojis in "🇦 🇧 🇨 🇩 🇪 🇫 🇬 🇭 🇮 🇯 🇰 🇱 🇲 🇳 🇴 🇵 🇶 🇷 🇸 🇹 🇺 🇻 🇼 🇽 🇾 🇿".split():
-    convert_dict[chr(counter)] = emojis
+    counter = 65
+    for emojis in "🇦 🇧 🇨 🇩 🇪 🇫 🇬 🇭 🇮 🇯 🇰 🇱 🇲 🇳 🇴 🇵 🇶 🇷 🇸 🇹 🇺 🇻 🇼 🇽 🇾 🇿".split():
+        convert_dict[chr(counter)] = emojis
     # 'B' is 66, 'C' is 67 etc
-    counter += 1
+        counter += 1
 
 # numerical emojis
-counter = 0
-for emojis in "⿠ ⿡ ⿢ ⿣ ⿤ ⿥ ⿦ ⿧ ⿨ ⿩".split():
-    convert_dict[counter] = emojis
-    counter += 1
+    counter = 48
+    for emojis in "⿠ ⿡ ⿢ ⿣ ⿤ ⿥ ⿦ ⿧ ⿨ ⿩".split():
+        convert_dict[chr(counter)] = emojis
```

Comment:
```
somehow i cant get "counter = 0
for emojis in "⿠ ⿡ ⿢ ⿣ ⿤ ⿥ ⿦ ⿧ ⿨ ⿩".split():
convert_dict[counter] = emojis
counter += 1" to work.
```
This is why it does not work : you changed `convert_dict[counter] = emojis` to `convert_dict[chr(counter)] = emojis`. Which the ASCII characters from `chr(0)` to `chr(9)` are not what we want.

You can see it here: https://www.ascii-code.com/